ibm_catalog.json

```diff
@@ -131,6 +131,23 @@
                             }
                         ]
                     }
+                },
+                {
+                    "label": "Cross account KMS encryption example",
```

Comment:
remove the word example from here (it shows as example in catalog so no need to add it in title too)

Reply:
Done.